marc_to_solr/lib/traject_config.rb

```diff
@@ -1305,43 +1355,32 @@
 # Position 852|k in the beginning of the call_number_display
 # The call_number_display is used in the catalog record page.
 to_field 'call_number_display' do |record, accumulator|
-  MarcExtractor.cached('852hik').collect_matching_lines(record) do |field, _spec, _extractor|
-    accumulator << [field['k'], field['h'], [field['i']]].compact.join(" ")
+  MarcExtractor.cached('AVAd').collect_matching_lines(record) do |field, _spec, _extractor|
+    accumulator << field['d']
   end
 end
 
 # Position 852|k at the end of the call_number_browse_s
 # The call_number_browse_s is used in the call number browse page in the catalog
 to_field 'call_number_browse_s' do |record, accumulator|
-  MarcExtractor.cached('852hik').collect_matching_lines(record) do |field, _spec, _extractor|
-    accumulator << [field['h'], field['i'], [field['k']]].compact.join(" ")
+  MarcExtractor.cached('AVAd').collect_matching_lines(record) do |field, _spec, _extractor|
+    accumulator << field['d']
   end
 end
 
 # The call_number_locator_display is used in the 'Where to find it' feature in the record page,
 # when the location is firestone$stacks.
-to_field 'call_number_locator_display', extract_marc('852hi')
+to_field 'call_number_locator_display', extract_marc('AVAd')
 
 to_field 'electronic_portfolio_s' do |record, accumulator|
   fields = []
-  dates = []
-  embargoes = []
-  MarcExtractor.cached('951knx').collect_matching_lines(record) do |field, _spec, _extractor|
+  MarcExtractor.cached('AVEmsu').collect_matching_lines(record) do |field, _spec, _extractor|
     fields << field
```

Comment:
Same here. AVE is a tag that comes from the Alma api call. In the exported files the data enrichment has 951 field. Please reverse the change.